docs_versioned_docs/version-ros2humble/ros/installation/nvidia.mdx

```diff
@@ -0,0 +1,262 @@
+
+If the **Download folder** and **SDKs instakll folder** specified at the bottom of the window do not already exist on
+your computer you will be prompted to create them. If this happens, click the green **Create** button.
+
+<center>
+  <figure>
+    <img
+      src={require("./img/jetpack_create.png").default}
+      width="400"
+    />
+  </figure>
+</center>
+
+If you are running on Ubuntu you may be asked to enter your password to run some commands as `root`.
+
+### Wait for the download and installation to progress
+
+Wait for JetPack to downloading and install. Depending on your internet connection speed this may take some
+time.
+
+<center>
+  <figure>
+    <img
+      src={require("./img/jetpack_progress.png").default}
+      width="900"
+    />
+  </figure>
+</center>
+
+The download and installation progress is shown at the bottom of the screen. If you need to pause the process, you can
+press the green **Pause** button and resume the installation at a later time.
+
+## Flash the Jetson
+
+:::note
+
+Some Jetson computers skip this step. Please refer to the instructions for your specific computer.
+
+For example, **Forcecr** boards use a different set of steps, [documented here](https://www.forecr.io/blogs/installation/jetpack-6-x-installation-for-dsboard-ornx).
+
+:::
+
+After the download has proceeded sufficiently, SDK Manager will prompt you to flash your Jetson computer.
+
+<center>
+  <figure>
+    <img
+      src={require("./img/jetpack_flash.png").default}
+      width="900"
+    />
+  </figure>
+</center>
+
+Enter the username and password you intend to use to log into the Jetson computer. By default Clearpath Robotics uses
+the username `administrator` and the password `clearpath`.
+
+When ready, click the green **Flash** button at the bottom of the window.
+
+## Post-installation configuration
+
+:::note
+
+See Clearpath's [computer Jetson setup script documentation](https://github.com/clearpathrobotics/clearpath_computer_installer/blob/feature/humble-jetson/README.md)
+for mode details on post-installation configuration.
+
+:::
+
+Once the core OS and Jetson APIs are installed the next step is to install the necessary ROS components to allow the
+computer to control your robot.
+
+Connect the Jetson to the internet, log in, and run the following command to download and run Clearpath's
+Jetson setup script:
+
+```bash
+wget -c https://raw.githubusercontent.com/clearpathrobotics/clearpath_computer_installer/refs/heads/feature/humble-jetson/jetson-humble-setup.bash && bash -e jetson-humble-setup.bash
+```
+
+This script will:
+- (re-)configure `apt` with addional sources;
+- install additional `apt` and `pip` packages needed for Clearpath robots;
+- install core ROS components;
+- create and build a ROS workspace (`$HOME/colcon_ws`);
+- install additional Linux kernel modules;
+- configure your robot's initial `/etc/clearpath/robot.yaml` file; and
+- optionally reconfigure carrier board USB connections to enable bluetooth.
```

Comment:
Add that bluetooth is needed for tele-op.

Reply:
Done.